src/App.js

```diff
@@ -1,30 +1,65 @@
-import { useEffect, useState } from "react"
-import { Link, Route, Routes } from "react-router-dom"
-import ContactsList from "./components/ContactsList"
-import ContactsAdd from "./components/ContactsAdd"
-import ContactsView from "./components/ContactsView"
-import "./styles/styles.css"
+import { useEffect, useState } from "react";
+import { Link, Route, Routes } from "react-router-dom";
+import { useFetch } from "./hooks/useFetch";
```

Comment:
Same as @AlexDjangoX  – did you find it useful to use ```useFetch```hook? I'm going to look into it....